Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/OverlayMessagingService.java

```diff
@@ -0,0 +1,125 @@
+package in.juspay.mobility.utils;
+
+import android.app.Service;
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.graphics.PixelFormat;
+import android.os.IBinder;
+import android.provider.Settings;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import androidx.annotation.Nullable;
+import com.bumptech.glide.Glide;
+import org.json.JSONObject;
+import in.juspay.mobility.R;
+
```

Comment:
Can you add a template of payload to send through FCM as comment in this file here?

Reply:
Added the same